src/demo/previewer.tsx

```diff
@@ -1,14 +1,19 @@
 import { h, render } from 'preact'
 import { memo } from 'preact/compat'
 import { useEffect, useRef, useState } from 'preact/hooks'
-import { getInitSdkOptions } from './demoUtils'
+
+import { ExtendedWindow } from '~types/global'
+import { SdkOptions } from '~types/sdk'
+
+import { CheckData, ViewOptions, getInitSdkOptions } from './demoUtils'
 import {
-  SdkOptions,
-  ViewOptions,
-  CheckData,
+  SdkOptionsView,
+  ViewOptionsView,
+  CheckDataView,
   SystemInfo,
 } from './SidebarSections'
 
+const extendedWindow = window as ExtendedWindow
```

Comment:
Here also, not a fan of renaming `window`.

What if you do something like:

```typescript
declare global {
  interface Window {
    updateOptions: ...
  }
}
```

Reply:
Thanks, @josokinas. Last week I tried that solution but it didn't work, that's why I used the ugly `ExtendedWindow`. Luckily today I've resolved that problem and made a global declaration to extend the `Window` interface itself.

Reply:
Lovely!